test/plugin/run.sh

```diff
@@ -140,11 +140,19 @@ if [[ ! -d ${agent_home} ]]; then
     echo "[WARN] SkyWalking Agent not exists"
     ${mvnw} --batch-mode -f ${home}/../../pom.xml -Pagent -DskipTests clean package
 fi
+# if it fails last time, relevant information will be deleted
+sed -i '/<sourceDirectory>scenarios\/'"$scenario_name"'<\/sourceDirectory>/d' ./pom.xml
+# add scenario_name into plugin/pom.xml
+echo check code with the checkstyle-plugin
+sed -i '/<\/sourceDirectories>/i <sourceDirectory>scenarios\/'"$scenario_name"'<\/sourceDirectory>' ./pom.xml
+
```

Comment:
You inject the style check in the CI control file, what is the purpose of this?

Reply:
It ensures that the code will be checked when we run the scenario locally.

Reply:
I see. But I feel a little concern about the dirty config(added but not removed somehow) will be pushed.

Reply:
I check out the newest code, 8.5.0-SNAPSHOT.
Run this script failed in my MacBook. How to fix it?
ref: [local-test-and-pull-request-to-the-upstream](https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md#local-test-and-pull-request-to-the-upstream)

```bash
➜  skywalking git:(master) bash ./test/plugin/run.sh hbase-scenario
+++ dirname ./test/plugin/run.sh
++ cd ./test/plugin
++ pwd
+ home=/Users/nisiyong/CodeProjects/skywalking/test/plugin
+ scenario_name=
+ force_build=off
+ cleanup=off
+ debug_mode=
+ mvnw=/Users/nisiyong/CodeProjects/skywalking/test/plugin/../../mvnw
+ agent_home=/Users/nisiyong/CodeProjects/skywalking/test/plugin/../../skywalking-agent
+ jacoco_home=/Users/nisiyong/CodeProjects/skywalking/test/plugin/../jacoco
+ scenarios_home=/Users/nisiyong/CodeProjects/skywalking/test/plugin/scenarios
+ num_of_testcases=
+ image_version=jdk8-1.0.0
++ date +%s
+ start_stamp=1616572417
+ parse_commandline hbase-scenario
+ test 1 -gt 0
+ _key=hbase-scenario
+ case "$_key" in
+ scenario_name=hbase-scenario
+ shift
+ test 0 -gt 0
+ [[ off == \o\n ]]
+ test -z hbase-scenario
+ [[ ! -d /Users/nisiyong/CodeProjects/skywalking/test/plugin/../../skywalking-agent ]]
+ sed -i '/<sourceDirectory>scenarios\/hbase-scenario<\/sourceDirectory>/d' ./pom.xml
sed: 1: "./pom.xml": invalid command code .
```